sources/zsh/aliases.d/ligolo-ng

```diff
@@ -0,0 +1 @@
+alias ligolo-proxy="/opt/tools/ligolo-ng/proxy"
```

Comment:
There is a missing new line at the end of the file

Reply:
Should be okay as-is, we have a function that cleans things up